### PR TITLE
Rust Implementation for neuralnetwork0.py

### PR DESCRIPTION
**Pull request by TejasGupta-27**
Have added implementation in Rust for neuralnetwork0

### CODE REVIEW

File: Rust/neural_network0/src/main.rs
Comment (extinctsion) [inline Rust/neural_network0/src/main.rs:1]: Great but can you pls rename this file as neural_network0.rs ? As it will be more suitable name

**Review by TejasGupta-27 — COMMENTED** [review]
Completed

File: Rust/neural_network0/src/main.rs
Comment (extinctsion) [reply Rust/neural_network0/src/main.rs:1]: okay

**Review by extinctsion — APPROVED** [review]
done

**Pull request merged**